src/views/view_sycl.hpp

```diff
@@ -218,17 +218,23 @@ struct MatrixView<
   // Information related to the data
   container_t data_;
   // Information related to the operation
-  const index_t sizeR_;  // number of rows
-  const index_t sizeC_;  // number of columns
-  const index_t sizeL_;  // size of the leading dimension
-  const index_t disp_;   // displacementt od the first element
+  const index_t sizeR_;   // number of rows
+  const index_t sizeC_;   // number of columns
+  const index_t sizeL_;   // size of the leading dimension
+  const index_t stride_;  // internal stride between same row/column elements
```

Comment:
See my comment above regarding `stride` vs `inc`. I think it would be good if we can replace `stride_` with `inc_` here and all other instances where we have used `stride_` as it may be misleading.